Changes.md

```diff
@@ -179,6 +179,11 @@ Build
 1.2.x.x (relative to 1.2.9.0)
 =======
 
+Improvements
+------------
+
+- ImageReader: promoted 'fileValid' from the internal OpenImageIOReader that reports per frame whether that files exists or not.
+- OpenImageIOReader: Added 'fileValid' Bool plug that returns a value per frame whether that files exists or not.
```

Comment:
This is in the `1.2.x` section, but the PR is to `main`, which is where we're landing stuff for `1.3`. I think you probably want to retarget the PR to `1.2_maintenance` - alternatively you could move this into the `1.3` section above.

Reply:
ah okay, missed the spot, it's not super high priority so happy to move it up to just 1.3